### ISSUE TITLE
Suspicious viscosity parameters for solvents

### ISSUE THREAD

**Issue by jonwzheng** [ReactionMechanismGenerator/RMG-database#641]
### Overview
Numerous viscosity correlations in the RMG database are suspicious / lead to strange viscosity values, see (by solvent order):
197: [N,N-dibutylbutan-1-amine](https://rmg.mit.edu/database/solvation/libraries/solvent/197/)
169: [ethane-1,2-diol](https://rmg.mit.edu/database/solvation/libraries/solvent/169/)
161: [2-(2-hydroxyethoxy)ethanol](https://rmg.mit.edu/database/solvation/libraries/solvent/161/)
139: [pentane-1,5-diol](https://rmg.mit.edu/database/solvation/libraries/solvent/139/) (to a lesser extent)
138: [butane-1,4-diol](https://rmg.mit.edu/database/solvation/libraries/solvent/138/) (to a lesser extent)
137: [propane-1,3-diol](https://rmg.mit.edu/database/solvation/libraries/solvent/137/) (to a lesser extent)
133: [1,1,2,2-tetrabromoethane](https://rmg.mit.edu/database/solvation/libraries/solvent/133/)
111: [2-methoxy-2-methylbutane](https://rmg.mit.edu/database/solvation/libraries/solvent/111/)
104: [pentan-2-ol](https://rmg.mit.edu/database/solvation/libraries/solvent/104/)
102: [2-ethoxyethanol](https://rmg.mit.edu/database/solvation/libraries/solvent/102/)
98: [2-ethylhexan-1-ol](https://rmg.mit.edu/database/solvation/libraries/solvent/98/)
96: [4-methylpentan-2-ol <note, this is so high that it leads to an overflow error>](https://rmg.mit.edu/database/solvation/libraries/solvent/96/)
86: [1,2,4-trimethylbenzene](https://rmg.mit.edu/database/solvation/libraries/solvent/86/)
79: [butan-2-ol](https://rmg.mit.edu/database/solvation/libraries/solvent/79/)
76: [hexafluorobenzene](https://rmg.mit.edu/database/solvation/libraries/solvent/76/)

### Solution
Someone should double-check these values - I don't currently have access to the original print resource (Viswanath, D. S., Ghosh, T. K., Prasad, D. H. L., Dutt, N. V. K., and Rani, K. Y. (2007) Viscosity of Liquids. Springer, The Netherlands: Dordrecht.).
We can also just replace the suspicious solvents with values from DIPPR which are easily accessible to us and might be more accurate?
Are we also sure that the values from the Viswanath manuscript match the functional form used in RMG?

**Comment by jonwzheng** [comment]
It doesn't look like these are due to typos from parsing the digital reference. And the T values are in the valid T range. 
I wonder if there's an OCR issue? I requested a copy of the original hardcover to double check...

**Comment by jonwzheng** [comment]
The OCR is fine; these are just typos in the original print source.

In the Viswanath reference book, I found a list of **experimental data** at temperatures close to room temperature. Using these, I was able to figure out typos in the correlations, verifying that fixing them gives the same value as the temperatures.

For example, 79: [butan-2-ol](https://rmg.mit.edu/database/solvation/libraries/solvent/79/): the **A** parameter should be `-106.38` rather than `-10.638`. I will submit a PR that fixes these.
